transport/http_test.go

```diff
@@ -154,7 +155,7 @@ func newHTTPServer(t *testing.T) *httpServer {
 	})
 
 	go func() {
-		http.ListenAndServe(":10000", nil)
+		http.ListenAndServe(":10001", nil)
```

Comment:
this was causing panics because both servers were trying to run on the same port